### PR TITLE
UI - Add respec button to skills menu #VGM-766

### PR DESCRIPTION
**Pull request by veteran29**
- title

### CODE REVIEW

**Comment by SavageGameDesign** [comment]
Task linked: [VGM-766 Move respec from user actions to a button on the skills menu](https://app.clickup.com/t/6631910/VGM-766)

File: game/functions/core/client/displays/fn_displaySkills.sqf [hunk 1]
```diff
@@ -11,18 +11,27 @@ switch _mode do {
         _display setVariable ["vgm_currentSkillTree", createHashMap];
         _display setVariable ["vgm_currentSkill", createHashMap];
 
-        private _handlerId = ["vgm_skills_learnt", [_display, {
-            params ["", "_display"];
-            ["refreshUI", _display] call vgm_c_fnc_displaySkills;
-        }]] call para_g_fnc_event_subscribeLocal;
-        _display setVariable ["vgm_skills_ui_learntHandlerId", _handlerId];
+        private _refreshHandlersIds = [];
+        _display setVariable ["vgm_skills_ui_refreshHandlerIds", _handlerIds];
```
Comment (Spoffy) [inline game/functions/core/client/displays/fn_displaySkills.sqf]: Looks like we might be passing the wrong variable here?

**Review by Spoffy — CHANGES_REQUESTED** [review]
Flagged a possible minor bug, all good otherwise - looks perfect in-game.

File: game/functions/core/client/displays/fn_displaySkills.sqf (hunk 1 above)
Comment (veteran29) [reply game/functions/core/client/displays/fn_displaySkills.sqf]: Good catch, forgot to copy paste during refactoring.

**Pull request merged**